site/pmguides/src/paystack/paystack.md

```diff
@@ -0,0 +1,119 @@
+author: Andrew Nduati
+id: paystack
+summary: An introduction to Paystack's Postman Workspace
+categories: Getting-Started
+environments: web
+status: Draft
+feedback link: https://github.com/PaystackOSS
+tags: Getting Started, Developer
+
+# Get Started with the Paystack API
+
+<!-- ------------------------ -->
+
+## Overview
+
+Duration: 2
+
+Paystack (acquired by Stripe) is an online payment processor for merchants in Africa. Founded in Nigeria in 2016 we’ve expanded to several markets including Ghana, South Africa and Kenya. Developers and startups in these markets can get paid from anyone, anywhere in the world. Coupled with our comprehensive [Developer Docs](https://paystack.com/docs) and [API Reference](https://paystack.com/docs/api/), you can explore the various Paystack APIs, from Transactions to Transfers and Account Verification to Subscriptions.
+
+At the end of this guide, you'll have setup Paystack Collection and made your first test payment on Paystack.
+
+
+### Prerequisites
+1. [Sign up](https://paystack.com/signup) on Paystack and retrieve your **Test Secret API Key**. (See [guide for more](https://support.paystack.com/hc/en-us/articles/360009881600-Paystack-Test-Keys-Live-Keys-and-Webhooks))
+2. Visit the [Paystack Postman Workspace](https://www.postman.com/paystack-developers/workspace/paystack-api/overview)
+3. Open our [Developer Docs](https://paystack.com/docs) and [API Reference](https://paystack.com/docs/api/) to get more detailed explanations on the APIs we’ll cover
+
+
+### What You’ll Learn
+
++ Fork the Paystack collection on your Postman
```

Comment:
I would word this as, "Fork the Paystack collection to your workspace in Postman"